src/state/actions/getActiveDonationsForClient.ts

```diff
@@ -1,8 +1,8 @@
 import railsAxios from '@util/railsAxios';
 
 export const getActiveDonationsForClient = async store => {
-	const { jwt } = store.state;
-	const endpoint = '/donations/active';
+	const { jwt, user } = store.state;
+	const endpoint = `/donations/active?client_lat=${user.coords.latitude}&client_long=${user.coords.longitude}`;
```

Comment:
Hey Drew, I was trying to run this PR on my local machine and when I logged into the default "client@client.com" account I got a "Cannot read property 'latitude' of undefined" error from line 5 of getActiveDonationForClient.ts. I did pull the latest version of the backend with your PR # 96. Is there another login I should use to test this?

Reply:
Hey Joseph, how are you testing--web, expo app, or emulator?  I'm going to double check that things work for me and possibly add better error handling around retrieving the client's location.